Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV GITLAB_VERSION=${VERSION} \
     RUBY_VERSION=2.6 \
     GOLANG_VERSION=1.13.10 \
     GITLAB_SHELL_VERSION=13.2.0 \
-    GITLAB_WORKHORSE_VERSION=8.31.0 \
+    GITLAB_WORKHORSE_VERSION=8.31.1 \
```

Comment:
GITLAB_WORKHORSE_VERSION=8.32.1 \

Reply:
On the release branch https://gitlab.com/gitlab-org/gitlab-foss/-/blob/13-0-stable/GITLAB_WORKHORSE_VERSION  `8.31.1` is available.